<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startlosangeles</class>
 <widget class="QMainWindow" name="startlosangeles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_routes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_listofstadiums">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_totalDist">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Total Distance</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_buySouv">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy Souvenir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_visit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Visit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
